File: src/Addins/Eto.Addin.MonoDevelop/Templates/Files/FS/code/preview.dialog.eto.fs
namespace ${Namespace}

open System
open Eto.Forms
open Eto.Drawing

type ${EscapedIdentifier}Base () as this =
    inherit Panel ()

    member this.InitializeComponent() =
        base.Title <- "My ${EscapedIdentifier} dialog"
        base.Content <- new Label (Text = "Some Content")

File: src/Addins/Eto.Addin.MonoDevelop/Templates/Files/CS/code/preview.dialog.eto.cs
using System;
using System.Collections.Generic;
using Eto.Forms;
using Eto.Drawing;

namespace ${Namespace}
{	
	partial class ${EscapedIdentifier} : Dialog
	{	
		void InitializeComponent()
		{
			Title = "My ${EscapedIdentifier} dialog";

			Content = new StackLayout
			{
				Items =
				{
					new Label { Text = "Some content" },
				}
			};

			// buttons
			DefaultButton = new Button { Text = "OK" };
			PositiveButtons.Add(DefaultButton);

			AbortButton = new Button { Text = "C&ancel" };
			AbortButton.Click += (sender, e) => Close();
			NegativeButtons.Add(AbortButton);
		}
	}
}
